datamade/la-metro-councilmatic
# Show today's meetings in place of about text

## Overview

This PR checks whether there are meetings today and replaces the about text on the index page with a list of today's meetings if there are meetings today.

### Checklist


### Demo
This is what the relevant portion of the index page looks like currently:
<img width="1092" alt="Screen Shot 2021-02-23 at 4 21 33 PM" src="https://user-images.githubusercontent.com/48227264/108915948-331d6800-75f3-11eb-9151-f165d2e836ee.png">

This is what the page currently looks like if there are meetings scheduled for today:
<img width="1121" alt="Screen Shot 2021-05-19 at 12 23 23 PM" src="https://user-images.githubusercontent.com/48227264/118856864-0482f680-b89d-11eb-9240-0918b20e0dd2.png">

## Testing Instructions

 * on your local app, do a windowed scrape for a few events using `docker-compose run --rm scrapers pupa update lametro events --window=0.05 rpm=0`
 * bring the app up locally and go to the site homepage. Make sure you see a list of today's meetings replace the description text to the right of the "Next Board Meeting" section

Handles #694
